frontend/src/metabase/styled-components/selectors.ts

```diff
@@ -20,7 +20,9 @@ export const getFont = createSelector(
 export const getFontFiles = createSelector(
   [getSettings, getEmbedOptions],
   (settings, embedOptions) => {
-    if (embedOptions.font) {
+    // If a font is set in the embed options, we don't need to load any font files.
+    // However, we still need to load the custom font files if the font is set to "Custom".
+    if (embedOptions.font && embedOptions.font !== "Custom") {
```

Comment:
I'm adding a unit test today. I just added you as reviewer first, so I can confirm if this makes sense.